src/main/java/gregtech/api/recipes/RecyclingHandler.java

```diff
@@ -126,6 +156,22 @@ private static void addItemStackToMaterialStacks(@NotNull ItemStack itemStack,
         }
     }
 
+    private static void addFluidStackToMaterialStacks(@NotNull FluidStack fluidStack,
```

Comment:
Do we want to check that the fluid stack is also a Fluid GT Material, and not a Gas or a plasma GT Fluid Material?